1_fetch/src/fetch_data.R

```diff
@@ -0,0 +1,25 @@
+# Load necessary packages
+library(dplyr)
+library(readr)
+library(stringr)
+library(sbtools)
+library(whisker)
```

Comment:
Note that not all of these libraries are necessary to run `fetch_data()` - can you move these library calls so that each `{X_phase}/src/{script}.R` file loads exactly those libraries needed within that script? (This will be slight overkill for this particular example but I think will be good, quick practice for real projects with multiple scipiper .yml files.)

Reply:
I thought about that, but, as I'm not totally familiar with all of these packages, wasn't sure which to remove. Is there any quick way to check, besides commenting out lines and re-running the script?

Reply:
Hmm, good question. When you write your own code, it will be clearer to you which functions come from which packages. You _could_ run `?{function}` (e.g., `?str_remove_all`) for each function whose provenance you don't know - at the top of the help file it says which package the function comes from. But that seems like a lot to ask of you for this exercise...so let's just skip it. Leave them all at the top of each script and we'll move them to a consolidated location in the next PR anyway.